src/main/com/example4/radek/DrawMeSquareApplication.java

```diff
@@ -0,0 +1,90 @@
+package main.com.example4.radek;
```

Comment:
nevychazel jsi z pposledniho devu, takze si musis udelat merge nebo rebase s devem a musi to jit do example004

Reply:
V té době ještě byl aktuální, ale mrknu na ten git